src/cljfx/coerce.clj

```diff
@@ -517,3 +518,46 @@
   (case x
     :use-computed-size -1.0
     (double x)))
+
+(defn animation [x]
+  (cond
+    (= :indefinite x) Animation/INDEFINITE
+    :else (int x)))
+
+(defn interpolator [x]
```

Comment:
should be moved to `cljfx.fx.transition`

Reply:
Done.